spec/requests/landing_page_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
 
-describe 'Home page', type: :request do
+describe 'Landing page', type: :request do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.